demo/api/src/main.ts

```diff
@@ -38,6 +39,7 @@ async function bootstrap(): Promise<void> {
         }),
     );
 
+    app.use(json({ limit: "1mb" }));
```

Comment:
```suggestion
    app.use(json({ limit: "1mb" })); // increase default limit of 100kb for saving large pages
```

Should we add the same comment as in Starter?